modules/vaos/spec/services/middleware/vaos_errors_spec.rb

```diff
@@ -0,0 +1,156 @@
+# frozen_string_literal: true
```

Comment:
is there a reason this file path doesn't parallel the class's file path?

Reply:
i think we just did what worked, no reason. What would you recommend we move it to?